src/EulerHS/KVConnector/Flow.hs

```diff
@@ -796,7 +796,7 @@ findAllWithOptionsKVConnector dbConf meshCfg whereClause orderBy mbLimit mbOffse
       kvRes <- redisFindAll meshCfg whereClause
       case kvRes of
         Right kvRows -> do
-          let matchedKVLiveRows = fst kvRows
+          let matchedKVLiveRows = findAllMatching whereClause (fst kvRows)
```

Comment:
why to apply whereClause again.

Reply:
we have removed from the redisFindAll that's why we are using it here